src/commands/image/imageSource/buildImageInAzure/RunStep.ts

```diff
@@ -27,10 +29,21 @@ export class RunStep extends ExecuteActivityOutputStepBase<BuildImageInAzureImag
                 dockerFilePath: path.basename(context.dockerfilePath) /* Assume the dockerfile is always in the root of the source */
             };
 
-            const building: string = localize('buildingImage', 'Building image...');
-            progress.report({ message: building });
+            const retries = 3;
+            await retry(
+                async (currentAttempt: number): Promise<void> => {
+                    const message: string = currentAttempt === 1 ?
+                        localize('buildingImage', 'Building image...') :
+                        localize('buildingImageAttempt', 'Building image (Attempt {0}/{1})...', currentAttempt, retries + 1);
+                    progress.report({ message: message });
+                    ext.outputChannel.appendLog(message);
 
-            context.run = await context.client.registries.beginScheduleRunAndWait(context.resourceGroupName, context.registryName, runRequest);
+                    runRequest.sourceLocation = 'a'
```

Comment:
What's this all about?

Reply:
That is an oopsie by me. I was using this to make the run fail.